database/postgresql-jiva/dbconnection.js

```diff
@@ -3,7 +3,7 @@ var config = {
   user: "testuser",
   database: "userdb",
   password: "password",
-  host: "pgset.postgresql-jiva",
+  host: "pgset-primary.postgresql-jiva",
```

Comment:
it should be pgset.postgresql-jiva

Reply:
Pgset-primary is also service for postgresql

Reply:
it will not work, did you tested it

Reply:
No , i didnt test ! In local I can’t test by using service ! 

In deployment repo using this host name.!

I will change it to ‘pgset’ 👍👍